templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>Manufacturer List <a style="float: right" href="{% url 'taxi:manufacturer-create' %}">+</a>
```

Comment:
The inline style for the 'Create' link is used to float it to the right. Consider using a CSS class instead of inline styles for better maintainability and separation of concerns.